sentinel/src/schedule/index.js

```diff
@@ -34,8 +35,11 @@ exports.sendable = function(config, now) {
   return now >= after && now <= until;
 };
 
+let timeout;
+
 exports.checkSchedule = function() {
   const now = moment(date.getDate());
+  clearTimeout(timeout);
```

Comment:
This change is made to support the scheduler restart mentioned earlier. Will be removed once it's possible to restart sentinel in e2e tests.